dwa_local_planner/src/dwa_planner.cpp

```diff
@@ -431,6 +432,59 @@ namespace dwa_local_planner {
     velocity_costs_.setGoalDistanceSquared(sq_dist);
   }
 
+  bool DWAPlanner::getDistanceAndTimeToGo(const tf::Stamped<tf::Pose>& poseTf, double& distance, double& time)
```

Comment:
TimeEstimate?  TimeToGo sounds odd :)

Reply:
Yeah, that was kind of bad in retrospect.  Changing it.